db/index.js

```diff
@@ -6,30 +6,3 @@ const homes = mariadb.createPool({socketpath: '/run/mysqld/mysqld.sock', user: '
 module.exports = {
```

Comment:
Consider `module.exports = homes`. This is just a cosmetic choice.

Reply:
Resolved.